src/main/ipv-stub/ipv-authorize.ts

```diff
@@ -78,15 +79,18 @@ async function get(
 }
 
 async function post(
-  _event: APIGatewayProxyEvent
+  event: APIGatewayProxyEvent
 ): Promise<APIGatewayProxyResult> {
   const redirectUri = `${ROOT_URI}/ipv-callback`;
 
   const url = new URL(redirectUri);
   url.searchParams.append("code", AUTH_CODE);
 
+  const params = Object.fromEntries(new URLSearchParams(event.body!));
```

Comment:
I think I'd prefer to explicitly check for non-null rather than assume it here

Reply:
Will do!